### PR TITLE
Add svx files to prettier config

### PR DESCRIPTION
**Pull request by diepoe**
When the git hook should run over the `.svx` files, it didn't worked. This tells prettier, how to format the files, I used the `mdx` option for that.

### CODE REVIEW

**Comment by MacFJA** [comment]
If we have this issue with the precommit hook, but not when we run `npm run lint` or `npm run format` maybe it miss a configuration to tell Prettier to look at `.svx` too ? 🤔

**Comment by diepoe** [comment]
Yeah, true!

**Comment by diepoe** [comment]
ae2c8d0 also adds json files to the lint-staged config

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		"build": "svelte-kit build",
 		"preview": "svelte-kit preview",
 		"lint": "prettier --check . && eslint --ignore-path .gitignore .",
```
Comment (MacFJA) [inline package.json]: Shouldn't `lint` have `./**/*.{js,ts,css,md,svx,svelte,html,json}` too ?

Comment (diepoe) [reply package.json]: yes, let me fix that

**Review by brittneypostma — APPROVED** [review]

**Review by MacFJA — APPROVED** [review]

**Comment by diepoe** [comment]
I have to run `npm run format` for all of the svx files now, because they never got formatted...

**Comment by brittneypostma** [comment]
Looks like the lint check is failing now, can you check these out and run `npm run format`?

**Review by MacFJA — APPROVED** [review]

**Pull request merged**